### PR TITLE
MGDOBR-499: Fix wrong assertj calls order

### PR DESCRIPTION
**Pull request by jomarko**
[MGDOBR-499](https://issues.redhat.com/browse/MGDOBR-499)

Replacement of #549 

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by manstis — APPROVED** [review]

File: infra/src/test/java/com/redhat/service/bridge/infra/BridgeErrorServiceTest.java [hunk 1]
```diff
@@ -63,14 +65,22 @@ private void checkId(BridgeError bridgeError) {
...
     private void checkExceptionIsInCatalog(Class<?> clazz) {
         BridgeError bridgeError = service.getError(clazz).get();
-        assertThat(bridgeError).isNotNull().withFailMessage(String.format("exception %s not found in the errors", clazz));
-        assertThat(service.getUserError(bridgeError.getId()).isPresent()).isTrue().withFailMessage(String.format("exception %s not found in the user errors", clazz));
+        assertThat(bridgeError)
+                .withFailMessage(String.format("exception %s not found in the errors", clazz))
+                .isNotNull();
+        assertThat(service.getUserError(bridgeError.getId()).isPresent())
+                .withFailMessage(String.format("exception %s not found in the user errors", clazz))
+                .isTrue();
```
Comment (sutaakar) [inline infra/src/test/java/com/redhat/service/bridge/infra/BridgeErrorServiceTest.java:73]:
```suggestion
        assertThat(service.getUserError(bridgeError.getId()))
                .withFailMessage(String.format("exception %s not found in the user errors", clazz))
                .isNotEmpty();
```

File: infra/src/test/java/com/redhat/service/bridge/infra/BridgeErrorServiceTest.java [hunk 2]
```diff
@@ -63,14 +65,22 @@ private void checkId(BridgeError bridgeError) {
...
     private void checkExceptionIsNotInCatalog(Class<?> clazz) {
         BridgeError bridgeError = service.getError(clazz).get();
-        assertThat(bridgeError).isNotNull().withFailMessage(String.format("exception %s not found in the errors", clazz));
-        assertThat(service.getUserError(bridgeError.getId()).isPresent()).isFalse().withFailMessage(String.format("exception %s should not be in the user errors", clazz));
+        assertThat(bridgeError)
+                .withFailMessage(String.format("exception %s not found in the errors", clazz))
+                .isNotNull();
+        assertThat(service.getUserError(bridgeError.getId()).isPresent())
+                .withFailMessage(String.format("exception %s should not be in the user errors", clazz))
+                .isFalse();
```
Comment (sutaakar) [inline infra/src/test/java/com/redhat/service/bridge/infra/BridgeErrorServiceTest.java:83]:
```suggestion
        assertThat(service.getUserError(bridgeError.getId()))
                .withFailMessage(String.format("exception %s should not be in the user errors", clazz))
                .isEmpty();
```

**Comment by jomarko** [comment]
@sutaakar thank you for a review, proposed changes pushed.

**Review by sutaakar — APPROVED** [review]

**Comment by jomarko** [comment]
Tests failed as:
```
Run scacap/action-surefire-report@v1.0.10
Going to parse results form **/surefire-reports/TEST-*.xml
 **/failsafe-reports/TEST-*.xml
shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java:162 | com.github.tomakehurst.wiremock.client.VerificationException:  No requests exactly matched. Most similar request was:  expected:<
Not publishing test result due to skip_publishing=true
Error: There were 1 failed tests
```

**Pull request merged**